hexo.el

```diff
@@ -1153,21 +1153,36 @@ This is merely resonable for files in _posts/."
                             (hexo-find-command repo-path)
                             command-string))
 
+(defcustom hexo-shell-command-separator "&&"
+  "Default is &&, you may change it to `; and` in case you are using fish"
+  :group 'hexo
+  :type '(string))
+
+(defcustom hexo--server-run-command (format "hexo clean %s hexo generate %s hexo server --debug" hexo-shell-command-separator hexo-shell-command-separator)
```

Comment:
Totally wrong. You define this variable is a string list, but your default value is a string.